### DISCUSSION TITLE
Best settings for Device Tracker

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [travisghansen/hass-pfsense#146] [General]
Hello and thank you for this integration.
I switched to pfsense a couple of weeks ago, and this integration really helps monitoring everything.

However, I´m still struggeling with the device_tracker entities, for example my Android Fully Kiosk Nokia Smartphone. It jumps randomly from home to away and back. I know it has to do with not communicating with pfsense, although I thought android devices are always fetching something from the web. 
Anyone else found a solution or the right settings to keep a device "home" when it´s home?

![image](https://user-images.githubusercontent.com/41727850/219438895-e703ad05-145e-4dc9-a244-5127b33f256b.png)

Below are my integration settings. should these values be adjusted? Does it  help to adjust the pfsense settings for the ARP age as mentioned in the readme?

![image](https://user-images.githubusercontent.com/41727850/219439502-b5f508d5-aeb7-4a95-8aa8-467b728ecd18.png)

Thanks for help and keep up the good work,
Philipp

**Comment by travisghansen** [comment]
Hey! This fell through the cracks for me :( The consider home value should effectively be a multiple of the scan interval.

There are tradeoffs obviously for what values you set but it seems you’re after something more sticky than not. To achieve that I would suggest doing an interval of 30 seconds and 300 home time. Those settings said differently are: the device must be present 1 out of 10 checks when checking every 30 seconds…or the device must be seen once every 5 minutes checking in 30 second intervals..

Note that each time the device is ‘seen’ it effectively slides that 5 minute window.

A higher frequency interval makes the devices detected as ‘home’ quicker when in the physical world you are transitioning home. The higher consider home time makes away more delayed in the reverse scenario.
